src/monster.cpp

```diff
@@ -378,6 +378,16 @@ int monster::vision_range(const int x, const int y) const
     return range;
 }
 
+bool monster::sees(int &bresenham_slope, Creature *target) const
```

Comment:
This might be a bit confusing: all the other `sees` function have the parameter in reverse order: `player::sees(Creature, t)`, `Creature:sees` etc.